src/firebolt/service/types.py

```diff
@@ -37,74 +37,74 @@ class EngineStatus(Enum):
 
     ENGINE_STATUS_UNSPECIFIED = "ENGINE_STATUS_UNSPECIFIED"
 
-    # Logical record is created, however underlying infrastructure is not initialized.
-    # In other words this means that engine is stopped.
+    ''' Logical record is created, however underlying infrastructure is not initialized.
```

Comment:
These should probably be double quotes everywhere, since that is compliant with our linter, black: https://github.com/psf/black